compiler/README.md

```diff
@@ -0,0 +1,29 @@
+## Usage example
```

Comment:
Should we rename this containg folder? Compiler seems quite ambiguous?

Reply:
Yes, it is. I was using the same name as [Java](https://github.com/grpc/grpc-java/tree/master/compiler) and the [core](https://github.com/grpc/grpc/tree/master/src/compiler) repo. I've changed it to the name of protoc plugin, `protoc-gen-rust-grpc` now.